web/lib/api/proxy.ts

```diff
@@ -27,7 +35,7 @@ function getProxiedRequestHeaders(req: NextApiRequest, whitelist: string[]) {
   }
   // mqp: the backend uses this in the cloud armor rules to bypass GCP throttling
   result.append('X-Vercel-Proxy-Secret', process.env.VERCEL_PROXY_SECRET ?? '')
-  result.append('X-Forwarded-For', req.socket.remoteAddress || '')
+  result.append('X-Forwarded-For', '')
```

Comment:
If you deploy something like this please be careful not to leave this broken, this would break the GCP throttling rules.